packages/modules/fulfillment/src/services/fulfillment-module-service.ts

```diff
@@ -627,10 +604,43 @@ export default class FulfillmentModuleService<
     }
 
     return await this.baseRepository_.serialize<FulfillmentTypes.FulfillmentDTO>(
-      fulfillment,
-      {
-        populate: true,
-      }
+      fulfillment
+    )
+  }
+
+  @InjectManager("baseRepository_")
+  async createReturnFulfillment(
+    data: FulfillmentTypes.CreateFulfillmentDTO,
+    @MedusaContext() sharedContext: Context = {}
+  ): Promise<FulfillmentTypes.FulfillmentDTO> {
+    const { order, ...fulfillmentDataToCreate } = data
```

Comment:
Why do we have an order passed to the fulfillment module? I checked the typings and it seems it's an empty object now, but I guess it's better to remove it altogether?

Reply:
well I am not sure why we have that, it is also not used but it is required in the types and in the workflow for the fulfillment. I didn't want to do that in that pr until we know why we have that as I dont recall the need for it but I can still rm it everywhere once we are aware of why we added it in the first place. Also we have a link between order and fulfillment, so could it be for the provider?

cc @olivermrbl

Reply:
v1 uses the order to pass some info to the providers.
we could rename that to something else in the future though

Reply:
Yeah, what @carlos-r-l-rodrigues said. This is just a naming issue. The order can hold details that are required for the provider to create the fulfillment

Reply:
I can change that to `data` in my follow up PR where I already removed `order_id` from types.

Reply:
Tu e fulfillment already have a field data which is what is returned by the provider, we night need to think about something else that seems more meaningful ? extra_data?

Reply:
Or something more specific like provider_data?